pkg/client/client.go

```diff
@@ -55,6 +55,14 @@ func AddToScheme() error {
 var DefaultNewClient = NewClient
 
 func NewClient(token, apiEndpoint string) (runtimeclient.Client, error) {
+	return NewClientWitTransport(token, apiEndpoint, &http.Transport{
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true, // nolint: gosec
+		},
+	})
+}
+
+func NewClientWitTransport(token, apiEndpoint string, transport http.RoundTripper) (runtimeclient.Client, error) {
```

Comment:
```suggestion
func NewClientWithTransport(token, apiEndpoint string, transport http.RoundTripper) (runtimeclient.Client, error) {
```

Reply:
(small typo)

Reply:
Friday's morning a2a3c2f

Reply:
And thanks for the catch ;-)

Reply:
my pleasure! 😇

Reply:
can you click on approve now? or is there anything still missing?